### ISSUE TITLE
Dashboard does not work

### ISSUE THREAD

**Issue by maks1m** [abhinavsingh/proxy.py#557]
Dashboard does not work.

**To Reproduce**
1. Install proxy with `pip install --upgrade proxy.py`
2. Run `proxy.py` as 'proxy --enable-dashboard'
3. Do 'http://127.0.0.1:8899/dashboard/' to trigger error
4. See error in log
```
    HttpWebServerPlugin.read_and_build_static_file_response(
  File "/Users/<myusername>/opt/miniconda3/lib/python3.8/site-packages/proxy/http/server/web.py", line 93, in read_and_build_static_file_response
    with open(path, 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/<myusername>/opt/miniconda3/lib/python3.8/site-packages/proxy/public/dashboard/proxy.html'
```
I've checked folder for proxy and there are no `public` folder.

**Version information**
 - OS: MacOS BigSur 11.4
 - proxy.py Version 2.3.1
 - python version 3.8.5

**Comment by abhinavsingh** [comment]
Can you try these steps, sorry for some reasons this was never documented, probably because dashboard was left incomplete in few respects.

https://github.com/abhinavsingh/proxy.py/issues/324#issuecomment-609738365

**Comment by maks1m** [comment]
> Can you try these steps, sorry for some reasons this was never documented, probably because dashboard was left incomplete in few respects.
> 
> [#324 (comment)](https://github.com/abhinavsingh/proxy.py/issues/324#issuecomment-609738365)

Hi.
I've made all steps from the link above.
Launch 'python -m proxy --enable-dashboard'
And when I try to open `http://localhost:8899/dashboard/` nothing happens.
The page is not loaded, but also I don't see any errors in proxy output.
Also, I tried to open `http://localhost:8899/dashboard/proxy.html` also no luck.

**Comment by maks1m** [comment]
UPD.
Looks like I had hanged proxy processes.
I've restarted them and for now I get ```
FileNotFoundError: [Errno 2] No such file or directory: '/Users/maks/src/github/proxy.py/proxy/public/dashboard/proxy.html'
```
The issue is that `/proxy/` is redundant in this path and I don't know whether proxy itself tries to reach incorrect path or there is a mistake in symlink creation.

**Comment by abhinavsingh** [comment]
@maks1m I'll try this again at my end and update you.  Yes, symlink is necessary under current architecture.  But let me double check the steps and possibly also document them for future.

**Comment by Ristovski** [comment]
@abhinavsingh Any updates on this?

**Comment by abhinavsingh** [comment]
@Ristovski @maks1m I have updated instructions to run dashboard here https://github.com/abhinavsingh/proxy.py/pull/638. Symlink should not be necessary.

TL;DR:

1) make dashboard
2) proxy --threadless --enable-dashboard --static-server-dir dashboard/public

Give it a try via pull request.  This will be merged soon into the develop branch.

Thank you!!!

**Comment by Yusufkulcu** [comment]
> [@Ristovski](https://github.com/Ristovski) [@maks1m](https://github.com/maks1m) I have updated instructions to run dashboard here [#638](https://github.com/abhinavsingh/proxy.py/pull/638). Symlink should not be necessary.
> 
> TL;DR:
> 
> 1. make dashboard
> 2. proxy --threadless --enable-dashboard --static-server-dir dashboard/public
> 
> Give it a try via pull request. This will be merged soon into the develop branch.
> 
> Thank you!!!

I can't run the make dashboard command on my Windows Device. I get the error Make command not found. How do I solve what the problem may be. I can't run the dashboard either.
